### DISCUSSION TITLE
How to keep the footer navigation in English even if it is in Japanese

### DISCUSSION THREAD

**Discussion by js2ymmks** [squidfunk/mkdocs-material#4057] [Q&A] [answered]
hello

I have a problem with "Previous/Next" in the footer being converted to Japanese when Japanese is specified. I would like to keep this part in English, so please tell me how to do that.

I feel that it would be more natural to leave the navigation in English, without converting it to Japanese.

`language: ja`

**Comment by squidfunk** [comment, marked as answer]
You can use [custom translations](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#custom-translations), something like:

``` html
<!-- Import translations for language and fallback -->
{% import "partials/languages/ja.html" as language %}
{% import "partials/languages/en.html" as fallback %} 

<!-- Define custom translations -->
{% macro override(key) %}{{ {
  "footer.previous": "Previous",
  "footer.next": "Next"
}[key] }}{% endmacro %}

<!-- Re-export translations -->
{% macro t(key) %}{{
  override(key) or language(key) or fallback.t(key)
}}{% endmacro %}
```

**Reply by js2ymmks** [reply]
Thank you for your reply.
However, I could not get it to work. Could you please help me with the settings?
Is the following setting correct?

mkdocs.yml
```
theme:
  language: custom
```

overrides/partials/languages/custom.html
```
<!-- Import translations for language and fallback -->
{% import "partials/languages/ja.html" as language %}
{% import "partials/languages/en.html" as fallback %} 

<!-- Define custom translations -->
{% macro override(key) %}{{ {
  "footer.previous": "Previous",
  "footer.next": "Next"
}[key] }}{% endmacro %}

<!-- Re-export translations -->
{% macro t(key) %}{{
  override(key) or language(key) or fallback.t(key)
}}{% endmacro %}
```

**Reply by squidfunk** [reply]
I think you're missing:

```
theme:
  custom_dir: overrides
```

All this is explained in our [customization guide](https://squidfunk.github.io/mkdocs-material/customization/#setup-and-theme-structure), which I invite you to read in order to understand how customization, theme extension and overriding templates and blocks works.

**Comment by js2ymmks** [comment]
Thank you for your always prompt response.
As you suggested, I have tried and confirmed that the changes are made as expected.
When I rewrote the file saved in overrides directly and ran it, it was properly overwritten.

```
theme:
  name: material
  custom_dir: overrides
  language: ja
```

overrides\partials\languages\ja.html

```
  "footer.previous": "Previous",
  "footer.next": "Next",
```

On the other hand, with [the first method you recommended](https://github.com/squidfunk/mkdocs-material/discussions/4057#discussioncomment-3023004), I get an error when I run. I have not been able to figure out the cause of this for a long time. This method seemed smarter to me and I would like to master it.

Anyway, Now that I understand how to do the customization guide, let's assume that I have solved the problem for the moment.
Thank you for your cooperation.
